### ISSUE TITLE
GenericOAuthenticator.login_service no longer customizable in 16.0.0

### ISSUE THREAD

**Issue by dmpe** [jupyterhub/oauthenticator#643]
### Bug description

Using just recently released `3.0.0-beta.2`, setting `login_service:` option in helm chart does not seem to work anymore

```
hub:
  config:
    GenericOAuthenticator:
      .... 
      login_service: Keycloak
```

The log also says `JupyterHub configurable:200] Config option login_service not recognized by GenericOAuthenticator.  Did you mean client_secret?` which is clearly incorrect as `client_secret` is secret token for OAuth2.0

#### Expected behaviour

Would have been nice if we can have the ability to customize back ...

![image](https://github.com/jupyterhub/oauthenticator/assets/1799661/2a7615b0-0c50-4314-89e7-24eff6251136)

#### My setup

Helm chart on Kubernetes cluster, using this version of the helm chart that was released today https://jupyterhub.github.io/helm-chart/jupyterhub-3.0.0-beta.2.tgz

**Comment by consideRatio** [comment]
Ah, it isn't made configurable in the Authenticator base class https://github.com/jupyterhub/jupyterhub/blob/0e437224d087820cdf78e92e31ee75407d1db793/jupyterhub/auth.py#L214-L224.

We'll redefine it here to be configurable then, thanks for reporting this @dmpe!

**Comment by dmpe** [comment]
Thanks very much

**Comment by consideRatio** [comment]
@dmpe `[3.0.0-beta.2.git.6228.hfe344e7a](https://jupyterhub.github.io/helm-chart/jupyterhub-3.0.0-beta.2.git.6228.hfe344e7a.tgz) is available with oauthenticator 16.0.1 - does it work out better?
